src/Features/Core/Portable/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer_IncrementalAnalyzer.cs

```diff
@@ -429,5 +435,57 @@ private void RaiseProjectDiagnosticsRemoved(StateSet stateSet, ProjectId project
 
             RaiseDiagnosticsRemoved(projectId, nullSolution, stateSet, raiseEvents);
         }
+
+        private async Task FireAndForgetReportAnalyzerPerformanceAsync(Document document, CompilationWithAnalyzers analyzerDriverOpt, CancellationToken cancellationToken)
```

Comment:
:bulb: This can be named `ReportAnalyzerPerformanceAsync`. The usage is what makes this fire-and-forget, and it's clear in the caller due to the use of an async token.